### DISCUSSION TITLE
Registration

### DISCUSSION THREAD

**Discussion by hollyisbella** [IBAHCM/RPiR#73] [Archive]
Hi, please can I be added! :)

**Comment by richardreeve** [comment]
Okay, I've sent you an invitation - let me know if you have any problems.

**Reply by hollyisbella** [reply]
I haven't received the invitation (that I can see...), would you be able to try sending the invitation again?

**Reply by richardreeve** [reply]
Could you try going to https://github.com/orgs/IBAHCM and seeing if you see a button there to accept the invitation, or if not try here - https://github.com/orgs/IBAHCM/teams/rpir - the email should definitely have gone out, but if neither of those work, I'll try uninviting you and reinviting you or something!
